<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 215, 203);</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>511</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid rgb(251, 235, 149);
    color: rgb(251, 235, 149);
    padding: 10px 20px;
    border-radius: 20px;
}

QPushButton:hover {
    background-color:rgb(251, 235, 149);
    color: white;
}</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="result_credit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid rgb(251, 235, 149);
    color: rgb(251, 235, 149);
    padding: 10px 20px;
    border-radius: 20px;
}

QPushButton:hover {
    background-color:rgb(251, 235, 149);
    color: white;
}</string>
   </property>
   <property name="text">
    <string>result</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="anuitetni">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(95, 143, 156);</string>
   </property>
   <property name="text">
    <string>anuitetni</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(95, 143, 156);</string>
   </property>
   <property name="text">
    <string>differented</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="credit_amount">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="term">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="total_payout">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="overpayment">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="monthly_payment">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(95, 143, 156);</string>
   </property>
   <property name="text">
    <string>Credit amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(95, 143, 156);</string>
   </property>
   <property name="text">
    <string>Term</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(95, 143, 156);</string>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(95, 143, 156);</string>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(202, 177, 193);</string>
   </property>
   <property name="text">
    <string>Monthly payment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(202, 177, 193);</string>
   </property>
   <property name="text">
    <string>Overpayment on credit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>480</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(202, 177, 193);</string>
   </property>
   <property name="text">
    <string>Total payment</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
